verify.ts

```diff
@@ -189,9 +189,9 @@ const processContracts = async (): Promise<any> => {
           fs.writeFileSync(path.join(options.outputFlattened, `flattened.${contractName}.sol`), flattened);
         }
 
-        let version =
-          require(`${foundIn}/build/contracts/${contractName}.json`).compiler.version;
-        version = `v${version.substr(0, version.indexOf('Emscripten') - 1)}`;
+        const version = 'v0.5.17+commit.d19bba13';
+        //   require(`${foundIn}/build/contracts/${contractName}.json`).compiler.version;
```

Comment:
why is that ?
if the comment is not needed so please remove it

Reply:
The problem with the hardhat is that it doesn't provide the version in the compiled file like truffle did, so had to hardcode it, I'll just delete the comments

Reply:
deleted